File: VB/WindowsFormsApplication18/MRUComboBoxEditPopupListBoxViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors.Popup

Namespace MRUComboBox
	Public Class MRUComboBoxEditPopupListBoxViewInfo
		Inherits PopupListBoxViewInfo
		Public Sub New(ByVal owner As PopupListBox)
			MyBase.New(owner)
		End Sub

		Public Shadows ReadOnly Property Owner() As Object
			Get
				Return MyBase.Owner
			End Get
		End Property
	End Class
End Namespace

File: CS/WindowsFormsApplication18/MRUComboBoxEditPopupListBoxViewInfo.cs
using DevExpress.XtraEditors.Popup;

namespace MRUComboBox
{
	public class MRUComboBoxEditPopupListBoxViewInfo : PopupListBoxViewInfo
	{
		public MRUComboBoxEditPopupListBoxViewInfo(PopupListBox owner)
			: base(owner)
		{
		}

		public new object Owner
		{
			get
			{
				return base.Owner;
			}
		}
	}
}
